### PR TITLE
Adding option to discard value names.

### PR DESCRIPTION
**Pull request by DeepakRajendrakumaran**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -207,6 +207,7 @@ static void lPrintVersion() {
     printf("    [--print-target]\t\t\tPrint target's information\n");
     printf("    [--fuzz-test]\t\t\tRandomly perturb program input to test error conditions\n");
     printf("    [--fuzz-seed=<value>]\t\tSeed value for RNG for fuzz testing\n");
+    printf("    [--[no-]discard-value-names]\t\tDo not discard/Discard value names when generating LLVM IR.\n");
```
Comment (dbabokin) [inline src/main.cpp:210]: An option with [no-] should be sorted in alphabetical order as if [no-] is not present.
Could you sort the whole output for `devUsage()`? The sorting broken currently.

**Review by dbabokin — CHANGES_REQUESTED** [review]
Could you please rebase the PR and add this to newly added `tests/lit-tests/full_mask.ispc`, which also uses variable names and needs them to pass.

File: src/main.cpp (hunk 1 above)
Comment (DeepakRajendrakumaran) [reply src/main.cpp]: Done

**Review by dbabokin — APPROVED** [review]

**Comment by dbabokin** [comment]
LGTM

**Pull request merged**